src/main/scanLogic/scanManager.ts

```diff
@@ -240,7 +240,7 @@ export class ScanManager implements ExtensionComponent {
             this._logManager.logMessage('Applicability runner could not find binary to run', 'DEBUG');
             return {} as ApplicabilityScanResponse;
         }
-        let skipFiles: string[] = AnalyzerUtils.getApplicableExcludePattern(Configuration.getScanExcludePattern());
+        let skipFiles: string[] = AnalyzerUtils.getAnalyzerManagerExcludePattern(Configuration.getScanExcludePattern());
         this._logManager.logMessage(
             'Scanning directory ' + directory + ', for CVE issues: ' + Array.from(cveToRun.values()) + ', skipping files: ' + skipFiles,
```

Comment:
```suggestion
            "Scanning directory '" + directory + "' for CVE issues: " + Array.from(cveToRun.values()) + '. Skipping files: ' + skipFiles,
```
Let's do it also in scanIac, scanSecrets, and scanEos.